commands/types/types.go

```diff
@@ -79,7 +79,13 @@ type MasterDefinition struct {
 
 // MasterNodes defines an interface for configuring HA master nodes.
 type MasterNodes struct {
-	HighAvailability bool `yaml:"high_availability,omitempty"`
+	HighAvailability  bool             `yaml:"high_availability,omitempty"`
+	AvailabilityZones []string         `yaml:"availability_zones,omitempty"`
+	Azure             MasterNodesAzure `yaml:"azure,omitempty"`
```

Comment:
This should be a ptr

Reply:
What is a ptr?

Reply:
This is a note for myself, the PR is not up for review

Reply:
ptr = pointer